src/renderer/component/uriIndicator/view.jsx

```diff
@@ -3,7 +3,7 @@ import React from 'react';
 import Button from 'component/link';
 import { buildURI } from 'lbryURI';
 import classnames from 'classnames';
-import Icon from 'component/common/icon';
+// import Icon from 'component/common/icon';
```

Comment:
i assume you will remove the commented code here and elsewhere before we merge

Reply:
I wasn't planning on it. Mostly they are just reminders to come back to something. I'm not sure if I need to keep this, but I will know when I start on the File Page (next)

Reply:
sounds good